src/rez/cli/_entry_points.py

```diff
@@ -295,22 +267,35 @@ def run_rez_benchmark():
     return run("benchmark")
 
 
-@scriptname("rez-pkg-ignore")
+@register("rez-pkg-ignore")
 def run_rez_pkg_ignore():
-    check_production_install()
     from rez.cli._main import run
     return run("pkg-ignore")
 
 
-@scriptname("rez-mv")
+@register("rez-mv")
 def run_rez_mv():
-    check_production_install()
     from rez.cli._main import run
     return run("mv")
 
 
-@scriptname("rez-rm")
+@register("rez-rm")
 def run_rez_rm():
-    check_production_install()
     from rez.cli._main import run
     return run("rm")
+
+
+@register("_rez-install-test")
+def run_rez_install_test():
```

Comment:
This command is part of the integration tests. It allows us to introspect how rez was launched. It'll print a json with the path to the python executable, the arguments from `argv` and the interpreter flags (sys.flags).